### PR TITLE
Releasev1.7.0

### PR DESCRIPTION
**Pull request by baude**

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1197,7 +1197,7 @@ checksum = "e5ce46fe64a9d73be07dcbe690a38ce1b293be448fd8ce1e6c1b8062c9f72c6a"
 
 [[package]]
 name = "netavark"
-version = "1.7.0-dev"
+version = "1.7.0"
```
Comment (Luap99) [inline Cargo.lock:1200]: needs to be bumped to 1.8.0-dev as well

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## v1.7.0
+* Fix misleading dns disabled log
+* Dependency updates
```
Comment (Luap99) [inline RELEASE_NOTES.md:5]: There are a lot more chnages:
- `--config` is now required when dns is used
- netavark dhcp-proxy correctly renews the lease after dhcp time-out
- bridge: isolate=strict option has been added
- macvlan: bclim option has been added
- no_default_route option has been added
- static routes can now be configured

**Review by Luap99 — APPROVED** [review]
LGTM

**Comment by baude** [comment]
@Luap99 i'll upload artifacts and update cargo tomorrow morning

**Pull request merged**